core/modules/qdisp/Executive.cc

```diff
@@ -141,25 +137,6 @@ class Executive::DispatchAction : public util::VoidCallable<void> {
     ExecStatus& _status; ///< Reference back to exec status
 };
 
-class NotifyExecutive : public util::UnaryCallable<void, bool> {
```

Comment:
Prefer keeping NotifyExecutive inside .cc, but declaring static
factory function in .h
```
// .h: inside class Executive
static boost::shared_ptr<UnaryCallable<void, bool> >
newNotifier(Executive& e, int refNum);
// .cc
Executive::newNotifier(Executive& e, int refNum) {
 return NotifyExecutive::Ptr(new NotifyExecutive(e, refNum));
}
```
You can remove the static factory function inside NotifyExecutive then.